templates/taxi/car_detail.html

```diff
@@ -12,11 +12,30 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+    {% if user in car.drivers.all %}
```

Comment:
The condition `user in car.drivers.all` might not work as expected if `user` is not a `Driver` instance. Ensure that `user` is indeed a `Driver` or adjust the logic to handle different user types.